pkgs/tools/misc/cod/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
+  pname = "cod";
+  version = "unstable-2020-09-10";
```

Comment:
Technically there is a release: https://github.com/dim-an/cod/tree/v0.0.0

Reply:
not sure if we want to take it. 0.0.0 is a weird version number.